webapp/channels/src/components/logged_in/logged_in.tsx

```diff
@@ -133,7 +116,9 @@ export default class LoggedIn extends React.PureComponent<Props> {
 
         window.removeEventListener('focus', this.onFocusListener);
         window.removeEventListener('blur', this.onBlurListener);
-        window.removeEventListener('message', this.onDesktopMessageListener);
+
+        this.offDesktopListeners?.();
+        delete this.offDesktopListeners;
```

Comment:
I think this was here before. If you don't think this is necessary, we can remove it. 0/5

Reply:
I'd prefer to remove it if it doesn't really do anything. If nothing else, it'd be better to blank out the field by setting it to `undefined` rather than using `delete` since I think think those have slightly different semantics that I don't entirely understand